schema.sql

```diff
@@ -0,0 +1,24 @@
+DROP DATABASE IF EXISTS games;
+
+CREATE DATABASE games;
+
+\c games;
+
+DROP TABLE IF EXISTS gamesTable;
```

Comment:
I don't believe its necessary to drop a table for newly created database.  when you source that .sql (or .cql) file, dropping the database on the first line gets rid off all the associated tables too.

Reply:
I agree with you. I haven't looked into it but it kept erroring out when I didn't have that.